scripts/mobilize_sync.py

```diff
@@ -328,7 +355,8 @@ def mobilize_updates(mobilize_dict: dict, hq: list, hq_worksheet, hq_columns: di
             event_attendance_updates.append([status] + hq_row[hq_columns['date_joined']:
                                                               hq_columns['interest_form_responses']])
     # Send the updates to Hub HQ
-    hq_worksheet.update('E4:L', event_attendance_updates)
+    range = hq_column_letters['status'] + '4:' + hq_column_letters['days_since_last_attendance']
```

Comment:
Example of how we're referencing a dictionary now instead of stating the range directly